### DISCUSSION TITLE
Plea to restore HC character

### DISCUSSION THREAD

**Discussion by Cavebunny** [chromiecraft/chromiecraft#7219] [General]
Died due to bug at Flight master in Shattrath.
![WoWScrnShot_072024_193109](https://github.com/user-attachments/assets/f1f9193f-2993-40f9-9e92-2fc66d7b7b7c)
[
![WoWScrnShot_072024_192836](https://github.com/user-attachments/assets/349447b6-5044-4ac3-a9b5-bec2f17308d7)
](url)

Hope you can restore Kozi.

**Comment by r-o-b-o-t-o** [comment]
Your character has been revived.
Good luck for the end of your journey!

**Reply by Cavebunny** [reply]
Thanks!
